public/app/features/dashboard/components/ShareModal/ShareEmbed.tsx

```diff
@@ -73,32 +62,30 @@ export class ShareEmbed extends PureComponent<Props, State> {
   };
 
   render() {
-    const { useCurrentTimeRange, includeTemplateVars, selectedTheme, iframeHtml } = this.state;
+    const { useCurrentTimeRange, selectedTheme, iframeHtml } = this.state;
+    const isRelativeTime = this.props.dashboard ? this.props.dashboard.time.to === 'now' : false;
 
     return (
       <div className="share-modal-body">
         <div className="share-modal-header">
-          <Icon name="link" className="share-modal-big-icon" size="xxl" />
           <div className="share-modal-content">
-            <div className="gf-form-group">
-              <InlineField labelWidth={24} label="Current time range">
+            <FieldSet>
+              <Field
+                label="Current time range"
+                description={
+                  isRelativeTime ? 'Transforms the current relative time range to an absolute time range' : ''
+                }
+              >
                 <Switch
                   id="share-current-time-range"
                   value={useCurrentTimeRange}
                   onChange={this.onUseCurrentTimeRangeChange}
                 />
-              </InlineField>
-              <InlineField labelWidth={24} label="Template variables">
-                <Switch
-                  id="share-template-variables"
-                  value={includeTemplateVars}
-                  onChange={this.onIncludeTemplateVarsChange}
-                />
-              </InlineField>
-              <InlineField labelWidth={24} label="Theme">
+              </Field>
+              <Field label="Theme">
```

Comment:
Since the select has only 3 options available I suggest using [RadioButtonGroup](https://developers.grafana.com/ui/latest/index.html?path=/story/forms-radiobuttongroup--simple) component instead